payments/direct-debit/payments.md

```diff
@@ -0,0 +1,465 @@
+---
+title: Swedbank Pay Payments Direct Debit Payments
+sidebar:
+  navigation:
+  - title: Payments
+    items:
+    - url: /payments/
+      title: Introduction
+    - url: /payments/credit-account
+      title: Credit Account Payments
+    - url: /payments/credit-card
+      title: Credit Card Payments
+    - url: /payments/invoice
+      title: Invoice Payments
+    - url: /payments/direct-debit
+      title: Direct Debit
+    - url: /payments/direct-debit/payments
+      title: Direct Debit Payments
+    - url: /payments/mobile-pay
+      title: Mobile Pay Payments
+    - url: /payments/swish
+      title: Swish Payments
+    - url: /payments/vipps
+      title: Vipps Payments
+---
+
+{% include alert.html type="warning"
+                      icon="warning"
+                      header="Site under development"
+                      body="The Developer Portal is under construction and should not be used to integrate against Swedbank Pay's APIs yet." %}
+
+# Direct Debit Payment Pages
+
+>PayEx offer Direct debit bank payments for the Baltics (Estonia, Latvia and Lithuania).
+
+{:.table .table-striped}
+| **Country** | **Supported banks** |
+|![Estonia](https://developer.payex.com/xwiki/wiki/developer/download/Main/ecommerce/payex-payment-instruments/direct-debit-payments/direct-debit-payment-pages/WebHome/Estonia.png) Estonia | Swedbank |
```

Comment:
Please upload the images to GitHub and move the links down into the footer so the table becomes easier to read.